### ISSUE TITLE
Support `ordered` `insert`s 

### ISSUE THREAD

**Issue by AlekSi** [FerretDB/FerretDB#940]
After #1618 (It's needed to handle postgres IF NOT EXISTS issues).

When inserting documents with `ordered` set to `true` (which should be set as a default value), insert documents one-by-one in seperate transactions, skip invalid documents (for example, with `_id` values that are already present) and return errors just for them.

With `ordered` set to `false`, attempt to insert all documents in one transaction.

See https://docs.mongodb.com/manual/reference/method/db.collection.insertMany/

https://github.com/FerretDB/FerretDB/pull/1118

**Details:**

- Use `InTransactionRetry` if needed

**DoD:**

- [x] `ordered` param is supported
- [x] Check for both pg and Tigris. If Tigris doesn't work, create a separate issue.
- [x] Update supported commands doc - https://docs.ferretdb.io/reference/supported_commands/
- [x] Write compat tests. For example, we can attempt to insert a few docs with one invalid (e.g. with a regex as `_id` - see `InsertIDRegex` test case).

**Comment by noisersup** [comment]
I believe that this one is pretty similar to #200 and don't understand why do we have 2 issues for that.
